SKILL.tsv

```diff
@@ -5734,302 +5734,302 @@ SKILL_20151102_005733	$[Merkabah] petrifies [Cursed] enemies for 3 seconds.
+SKILL_20151224_005969	The enemies hit by [Slithering] is vulnerable to [Strike] for 10 seconds and increase the additional damage by 20% per attribute level.
+SKILL_20151224_005970	Targe Smash: Remove Knockback
+SKILL_20151224_005971	[Targe Smash] no longer knockback enemies.
+SKILL_20151224_005972	Increases your block penetration by 10% per attribute level when equipped with a [Two-handed Spear].
+SKILL_20151224_005973	Pistol Shot: Enhance
+SKILL_20151224_005974	Increases the damage dealt on an enemy with [Pistol Shot] by 1% per attribute level.
+SKILL_20151224_005975	Decreases damage taken to x2 from x3 when [Double Pay Earn] is active.
+SKILL_20151224_005976	Zonhau: Enhance
+SKILL_20151224_005977	Increases the damage dealt on an enemy with [Zonhau] by 1% per attribute level.
+SKILL_20151224_005978	Redel: Enhance
+SKILL_20151224_005979	Increases the damage dealt on an enemy with [Redel] by 1% per attribute level.
+SKILL_20151224_005980	Zucken: Enhance
+SKILL_20151224_005981	Increases the damage dealt on an enemy with [Zucken] by 1% per attribute level.
+SKILL_20151224_005982	Cyclone: Resist
+SKILL_20151224_005983	Increase the chance to block harmful effect by 15% per attribute level during [Cyclone].
+SKILL_20151224_005984	STA consumption cycle duration is increased by 0.05 seconds per attribute level when [Bunshin no Jutsu] is active.
+SKILL_20151224_005985	Reduce the damage taken by 4% per attribute level when there's least 2 clones from [Bunshin no Jutsu].
+SKILL_20151224_005986	Serpentine: Enhance
+SKILL_20151224_005987	Increases the damage dealt on an enemy with [Serpentine] by 1% per attribute level.
+SKILL_20151224_005988	Serpentine: Bleeding
+SKILL_20151224_005989	Afflicts [Bleeding] with 2% rate per attribute level of [Serpentine]for 8 seconds. Bleeding damage is proportional to the character's physical attack.
+SKILL_20151224_005990	Two-handed Spear Mastery: Large Target Hunter
+SKILL_20151224_005991	Increases the damage dealt on large enemies with [Two-handed Spear] by 4% per attribute level against normal enemies and 2% per attribute level against boss enemies.
+SKILL_20151224_005992	[Energy Bolt] damage is enhanced to Wizard circle 2 standard.
+SKILL_20151224_005993	[Energy Bolt] damage is enhanced to Wizard circle 3 standard.
+SKILL_20151224_005994	[Earthquake] damage is enhanced to Wizard circle 3 standard.
+SKILL_20151224_005995	Deals additional damage equal to 40% magic attack per attribute level when attacking an enemy under [Sleep] with [Energy Bolt].
+SKILL_20151224_005996	Enemies affected by [Lethargy] will receive 20% additional damage with Strike attacks.
+SKILL_20151224_005997	Increases magic damage by 10% per attribute level when [Quick Cast] is active.
+SKILL_20151224_005998	[Enchant Fire] will decrease the fire resistance of nearby enemies by 10 per attribute level for 5 seconds.
+SKILL_20151224_005999	Freeze duration of [Subzero Shield] is increased by 0.5 seconds per attribute level.
```

Comment:
Increases the freeze duration of [Subzero Shield] by 0.5 seconds per attribute level.